src/humanPose/draw.js

```diff
@@ -357,6 +357,7 @@ function toggleAnalyzerSettingsUI() {
  * @param {boolean} visible - whether to show or not
  */
 function setHumanPosesVisible(visible) {
+    if (!realityEditor.motionStudy.getAllHumanPoseAnalyzers) return;
```

Comment:
This whole function (and corresponding shortcut) can be removed. There's not much of a reason to provide a way to hide human poses when we also have a way to disable human pose tracking

Reply:
Ok, I'll remove those instead